editions/tw5.com/tiddlers/howtos/Using Stamp.tid

```diff
@@ -1,27 +1,59 @@
+
+Snippets can either ''replace'', or be ''added before and/or after'', the selected text in the editor.
 
 ! Create a snippet
-# Click ''stamp'' (<<.icon $:/core/images/stamp>>)
-# Create a snippet tiddler through the "//Add your own//" menu entry
-# Type some text as snippet for the tiddler, add a caption for the name as shown in the menu
-# Click the <<.icon $:/core/images/done-button>> ''ok'' button
-
-<$macrocall $name=".tip" _="""''Tip:'' You can also create a snippet tiddler using the ''new tiddler'' <<.icon $:/core/images/new-button>> button in the sidebar, and add tag ''~$:/tags/TextEditor/Snippet''""" />
-
-!!<<.from-version "5.1.20">> Adding a prefix and/or suffix to a selection
-
-# Click ''stamp'' (<<.icon $:/core/images/stamp>>)
-# Create a snippet tiddler through the "//Add your own//" menu entry
-# Add a caption for the name as shown in the menu
-# Create a tiddler with the same title but add the suffix `/prefix`
-# Insert the prefix in its text field
-# Create a tiddler with the same title but add the suffix `/suffix`
-# Insert the suffix in its text field
-# Click the <<.icon $:/core/images/done-button>> ''ok'' button
+!! Whilst editing a tiddler
+
+# Click ''stamp'' (<<.icon $:/core/images/stamp>>) on the toolbar
+# Click "//Add your own//" at the bottom of the menu
+# Type the text for the snippet in the editor
+# Enter the caption for the snippet menu item in the `caption` field
+# Click the <<.icon $:/core/images/done-button>> //Confirm changes to this tiddler// button
+
+!! Manually
+
+# Create a new tiddler by clicking the //Create a new tiddler// <<.icon $:/core/images/new-button>> button in the sidebar
+
+# Change the title for the tiddler (from e.g. `New Tiddler 1`). The tiddler title determines the menu item for the stamp if you do not set a `caption` field (see below).<div><$macrocall $name=".tip" _="""''Pro tip'' Set a title like `$:/yourusername/snippets/My new stamp` to 'file' your new tiddler away as a [[system tiddler|SystemTiddlers]] which doesn't appear in the normal search results """ /></div>
+
+# Add the tag <<tag-pill $:/tags/TextEditor/Snippet>> by entering `$:/tags/TextEditor/Snippet` in the `tag name` tag box and clicking //Add// (or pressing the //Enter// key)
+
+# Enter the snippet content (which you wish to appear in the editor when you click the stamp menu <<.icon $:/core/images/stamp>>) in the editor 
+
+# Create a field with the name `caption` and value set to the text for the snippet's stamp menu item:
+## Enter `caption` in the //Add new field// `field name` box
+## Enter the stamp menu item caption in the `field value` box
+
+
+
+! Re-ordering snippets
+Your new snippet will appear at the bottom of the stamp menu by default. To adjust the order of snippets:
+
+# Navigate to a snippet tiddler
+# Click the <<tag-pill $:/tags/TextEditor/Snippet>> tag pill
```

Comment:
hmm, It describes the "re-ordering" using the `<<tag` macro, but it doesn't work here. IMO the `<<tag` macro should be used. So reordering works here too.